client/TableSorting/Events.js

```diff
@@ -1,16 +1,15 @@
 //Events: click on Time button filter invoices
 Template.bTotal.events({
   'click button': function () {
-    Session.set('invoiceSortBy', 'total');
-    Session.set('invoiceSortOrder',Session.get('invoiceSortOrder')*(-1));
+    GoURL("createdAt");
   }});
 
 Template.bCreatedAt.events({
   'click button': function () {
-     Session.set('invoiceSortBy', 'createdAt');
-   Session.set('invoiceSortOrder',Session.get('invoiceSortOrder')*(-1));
+    GoURL("total");
   }});
 
- 
-
-
+function GoURL(sortedBy) {
```

Comment:
Please use `camelCase` (first character lower case) for all variables / functions / methods in Javascript!